### PR TITLE
added multilinugal top menu

### PR DESCRIPTION
**Pull request by critzo**
This PR adds logic for a top menu in multiple languages and google translate versions of the top menu in Spanish and French

### CODE REVIEW

File: commotionwireless.net/_includes/menu-navbar-main_fr.html [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+        <ul class="nav navbar-nav">
+            <li class="dropdown">
+                <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false"><span class="glyphicon glyphicon-plus"></span> Sur</a>
+                <ul class="dropdown-menu">
+                    <li><a href="{{site.baseurl}}/{{page.lang}}/about">À propos de Commotion</a></li>
+                    <li><a href="{{site.baseurl}}/{{page.lang}}/about/contributors">Commotion Contributeurs</a></li>
+                    <li><a href="{{site.baseurl}}/{{page.lang}}/about/guiding-principles">Des principes directeurs</a></li>
+                    <li><a href="{{site.baseurl}}/{{page.lang}}/about/where-its-used">Où il est utilisé</a></li>
+                    <li><a href="{{site.baseurl}}/{{page.lang}}/about/press">Presse</a></li>
+                    <li><a href="{{site.baseurl}}/{{page.lang}}/about/license-privacy">Licence &amp; Intimité</a></li>
```
Comment (georgiamoon) [inline commotionwireless.net/_includes/menu-navbar-main_fr.html]: `Intimite` should be `confidentialité`

File: commotionwireless.net/_includes/menu-navbar-main_fr.html [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+                    <li><a href="{{site.baseurl}}/{{page.lang}}/about/funding">Financement</a></li>
+                    <li><a href="{{site.baseurl}}/{{page.lang}}/jobs">Emplois</a></li>
+                    <li><a href="{{site.baseurl}}/{{page.lang}}/contact">Contact</a></li>
+                </ul>
+            </li>
+            <li><a href="{{site.baseurl}}/{{page.lang}}/blog">Blog</a></li>
+            <li class="dropdown">
+                <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false"><span class="glyphicon glyphicon-plus"></span> Documentation</a>
```
Comment (georgiamoon) [inline commotionwireless.net/_includes/menu-navbar-main_fr.html]: there's a stray space here before `Documentation`

File: commotionwireless.net/_includes/menu-navbar-main_fr.html [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+            <li class="dropdown">
+                <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false"><span class="glyphicon glyphicon-plus"></span> Pour les développeurs</a>
+                <ul class="dropdown-menu">
+                    <li><a href="{{site.baseurl}}/{{page.lang}}/developer/resources">Pour les développeurs</a></li>
+                    <li><a href="https://github.com/opentechinstitute">Code source</a></li>
+                    <li><a href="{{site.baseurl}}/{{page.lang}}/developer/api">Référence de l'API</a></li>
+                    <li><a href="https://wiki.commotionwireless.net">Wiki</a></li>
+                    <li><a href="https://github.com/issues?q=is:issue+user:opentechinstitute+is:open">Suivi du problème</a></li>
+                    <li><a href="{{site.baseurl}}/{{page.lang}}/developer/hig/introduction">Guide de l'interface humaine</a></li>
+                    <li><a href="{{site.baseurl}}/{{page.lang}}/jobs">Requests for Proposals</a></li>
```
Comment (georgiamoon) [inline commotionwireless.net/_includes/menu-navbar-main_fr.html]: Should this also be `Emplois` ?

**Comment by georgiamoon** [comment]
Put some comments on the french translations.

File: commotionwireless.net/_includes/menu-navbar-main_fr.html (hunk 3 above)
Comment (critzo) [reply commotionwireless.net/_includes/menu-navbar-main_fr.html]: We had made this separate menu item 'requests for proposals' in the 'developer' section but it does link to the same place. I'm adding a translation for it now.

**Comment by critzo** [comment]
I am working on the rest of issue #360 so please hold off on merging until that's done.

**Comment by critzo** [comment]
@jheretic Multilingual menu support should now be complete with my last commit.

**Pull request closed without merging**